<!DOCTYPE html>

<html lang="en">
	<head>
		<!-- Global site tag (gtag.js) - Google Analytics -->
		<script async src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID"></script>
		<script>
			window.dataLayer = window.dataLayer || [];
			function gtag() { dataLayer.push(arguments); }
			gtag('js', new Date());

			gtag('config', 'UA-150363640-2');
		</script>
		<meta charset="UTF-8">
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
		<meta name="description" content="Roger Lefort's resume.">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">

		<!-- Boostrap -->
		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
		<link href="./static/css/bootstrap.min.css" rel="stylesheet" type='text/css'>
		<link href="./static/css/bootstrap-theme.min.css" rel="stylesheet" type='text/css'>

		<!-- Our css -->
		<link href="./static/css/resume.css" rel='stylesheet' type='text/css'>

		<!-- Fonts -->
		<link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
		<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,900" rel="stylesheet">
		<link href='https://fonts.googleapis.com/css?family=Lobster|Inconsolata:400,700|Open+Sans:400,300,700,600' rel='stylesheet' type='text/css'>

		<!-- jQuery -->
		<script type="text/javascript" src="https://code.jquery.com/jquery-3.1.0.js"></script>

		<title>Roger Lefort's resume</title>

	</head>

	<body style="zoom: 1;">
		<div class="container">
			<header>
			<div id="headshot">
				<img src="./static/Octocat.png" alt="Roger Lefort's headshot">
			</div>

			<div id="contactbox">
				<br/>
				<b id="name">Roger Lefort, PhD</b><br/>
				<b id="job-title">Neuroscientist | Data Scientist</b><br/>
				<br/>
				<script>
				var username = "roger";
				var hostname = "cyclingscientist.com";
				var linktext = username + "@" + hostname ;
				document.write("<a href='" + "mail" + "to:" + username + "@" + hostname + "'>" + linktext + "</a><br/>");
				</script>
				<!-- <a href="http://www.cyclingscientist.com">www.cyclingscientist.com</a><br/> -->
			</div>

			</header>

			<section>
			<h1>About me</h1>
			<br>
			<div>
				<h2>Summary</h2>
				<h3> I am a Neuroscientist, data wrangler and avid cyclist with 10+ years of experience spearheading research on central nervous system disorders. I have demonstrated expertise in rigorous experimental research design, data collection, statistical analyses and complex data interpretation and visualization. I am highly experienced in building and leading cross-functional teams and managing multidisciplinary projects. I have excellent interpersonal, communication and presentation skills honed through invited lectures and seminars, and preparation and publications of numerous peer-reviewed papers.</h3>

				<h3> Learning is what drives me — I believe that there is always room for improvement, especially with regards to my own abilities, and I constantly seek to sharpen my skills.</h3>
			</div>

			</section>
			<hr>
			<section>
		<article>

			<h2>Timeline</h2>
			<h4>My career path so far... Click on the timeline for more datails.</h4>

			<div id="timeline"></div>

		</article>

				<hr>
		<article>
			<h2>List of skills</h2>
			<div class="container">
				<div class="row">
					<div class="col-sm">
						<span id="skillset"><img class="lab" src="./static/brain_1.svg" alt="brain" width=24 height=24/><b id="subtitle">Neuroscience</b><br/><p id="skill-description"> I have over a decade of experience in CNS disease research, with a specific focus on Alzheimer's disease. I specialize in <i>in vivo</i>, <i>in vitro</i> and <i>ex vivo</i> disease modeling, using a variety of tools, including transgnenic mice and cell-derived models (primary and ESC/iPSC). I am highly skilled in biochemistry and molecular biology techniques, and data analysis and interpretation. I also have deep knowledge of the scientific literature and molecular signaling.</p></span>
					</div>
					<div class="col-sm">
						<span id="skillset"><img class="lab" src="./static/data_science.svg" alt="chart" width=24 height=24/><b id="subtitle">Data Science</b><br/> I am a Machine Learning enthusiast. I care about things like statistical significance and reproducibility. I am no graphic artist, but I like to draw stuff, make pretty graphs and presentations, and mess with colors and line widths. Data is no good without visualization. While my PhD is not in Data Science, I am deeply passionate about it.</span>
					</div>
					<div class="col-sm">
						<span id="skillset"><img class="lab" src="./static/prompt.svg" alt="prompt" width=24 height=24/><b id="subtitle">Programming</b><br/> My language of choice is Python, and its numerous data science libraries (pandas, numpy, scipy, keras, scikit, etc). I can also work with HTML/CSS, Javascript and SQL. My skills are more adapted for rapid prototyping and data analysis than production quality code. By the way, this resume is built using HTML, Bootstrap, CSS and Javascript D3, served through a flask server.</span>
					</div>
				</div>
			</div>

				<div id="listofskills"><h3><i>Other skills</i></h3</div>

		</article>

			<hr>

			<div class="page-break"></div><article><div id="experience"><h2>Experience</h2></div></article>
			<div class="page-break"></div><article><div id="education"><h2>Education</h2></div></article>
			<div class="page-break"></div><article><div id="other"><h2>Continuing Education</h2></div></article>
			<article>

			<hr>
		<article>
			<div id="publications">
			<h2>Select Publications</h2>
			<div class="container">
			<div class="row">
				<div class="col">
					<div id="wrapper">
						<div id="content">
							<img class="research" src="./static/research.svg" alt="research" width=5 height=5/><h4 id="manuscript-title">Type 1 Interleukin-4 Signaling Obliterates Mouse Astroglia in vivo but Not in vitro.</h4><p>Mashkaryan V, Siddiqui T, Popova S, Cosacak MI, Bhattarai P, Brandt K, Govindarajan M, Petzold P, Reinhardt S, Dahl A, Lefort R, Kizil C.</p>
							<a href="https://www.frontiersin.org/articles/10.3389/fcell.2020.00114/full" target="_blank" rel="noopener noreferrer">Front. Cell. Dev. Biol. 2020 Feb 26;8(114)</p></a>
							<p class="collapse" id="man7">Recent findings suggest that reduced neurogenesis could be one of the underlying reasons for the exacerbated neuropathology in humans, thus restoring the neural stem cell proliferation and neurogenesis could help to circumvent some pathological aspects of Alzheimer's disease. We recently identified Interleukin-4/STAT6 signaling as a neuron-glia crosstalk mechanism that enables glial proliferation and neurogenesis in adult zebrafish brain and 3D cultures of human astroglia, which manifest neurogenic properties. In this study, by using single cell sequencing in the APP/PS1dE9 mouse model of AD, we found that IL4 receptor (Il4r) is not expressed in mouse astroglia and IL4 signaling is not active in these cells. We tested whether activating IL4/STAT6 signaling would enhance cell proliferation and neurogenesis in healthy and disease conditions. Lentivirus-mediated expression of IL4R or constitutively active STAT6VT impaired the survival capacity of mouse astroglia in vivo but not in vitro. These results suggest that the adult mouse brain generates a non-permissive environment that dictates a negative effect of IL4 signaling on astroglial survival and neurogenic properties in contrast to zebrafish brains and in vitro mammalian cell cultures. Our findings that IL4R signaling in dentate gyrus (DG) of adult mouse brain impinges on the survival of DG cells implicate an evolutionary mechanism that might underlie the loss of neuroregenerative ability of the brain, which might be utilized for basic and clinical aspects for neurodegenerative diseases.</p>
							<p><a class="btn" data-toggle="collapse" data-target="#man7">Abstract &raquo;</a></p>
						</div>
					</div>
				</div>
				<div class="verticalLine"></div>
				<div class="col">
				<div id="wrapper">
					<div id="content">
						<img class="research" src="./static/research.svg" alt="research" width=5 height=5/><h4 id="manuscript-title">Alzheimer’s disease-associated P460L mutation in ephrin receptor type A1 (EphA1) leads to dysregulated Rho-GTPase signaling.</h4><p>Kim Y, Lasso G, Patel H, Vardarajan B, Santa-Maria I, Lefort R.</p>
						<a href="https://www.biorxiv.org/content/10.1101/2021.06.17.448790v1" target="_blank" rel="noopener noreferrer">bioRxiv 2021.06.17.448790 </p></a>
						<p class="collapse" id="man8">Recently, late onset AD (LOAD) genome-wide association studies identified EphA1, a member of receptor tyrosine kinase family (RTK) as a disease associated loci. In the follow-up study where 3 independent LOAD cohorts were performed, a P460L coding mutation in EphA1 loci showed a significant association with LOAD. However, the role of EphA1 and P460L mutant EphA1 in AD is not fully understood. We have characterized this mutation biophysically and biochemically. Our structural in silico model and in vitro biochemical analysis demonstrate that EphA1-P460L mutation makes the receptor constitutively active suggesting a gain-of-toxic function leading to chronic EphA1 signaling in the brain. Moreover, we report that the EphA1 P460L variant triggers Rho-GTPase signaling dysregulation that could potentially contribute to spine morphology abnormalities and synaptic dysfunction observed in AD pathology.</p>
						<p><a class="btn" data-toggle="collapse" data-target="#man8">Abstract &raquo;</a></p>
					</div>
				</div>
			 </div>
			</div>
		</div>
			<div class="container">
		  <div class="row">
		    <div class="col">
					<div id="wrapper">
						<div id="content">
							<img class="research" src="./static/research.svg" alt="research" width=5 height=5/><h4 id="manuscript-title">Dysfunction of the ubiquitin ligase E3A Ube3A/E6-AP contributes to synaptic pathology in Alzheimer’s disease.</h4><p>Olabarria M, Pasini S, Corona C, Robador P, Song C, Patel H, Lefort R.</p>
							<a href="https://www.nature.com/articles/s42003-019-0350-5" target="_blank" rel="noopener noreferrer">Commun. Biol. 2019 Mar 22;2(111)</p></a>
							<p class="collapse" id="man1">Synaptic dysfunction and synapse loss are prominent features in Alzheimer's disease. Members of the Rho-family of guanosine triphosphatases, specifically RhoA, and the synaptic protein Arc are implicated in these pathogenic processes. They share a common regulatory molecule, the E3 ligase Ube3A/E6-AP. Here, we show that Ube3A is reduced in an Alzheimer's disease mouse model, Tg2576 mouse, which overexpresses human APP695 carrying the Swedish mutation, and accumulates Aβ in the brain. Depletion of Ube3A precedes the age-dependent behavioral deficits and loss of dendritic spines in these mice, and results from a decrease in solubility following phosphorylation by c-Abl, after Aβ exposure. Loss of Ube3A triggers the accumulation of Arc and Ephexin-5, driving internalization of GluR1, and activation of RhoA, respectively, culminating in pruning of synapses, which is blocked by restoring Ube3A. Taken together, our results place Ube3A as a critical player in Alzheimer's disease pathogenesis, and as a potential therapeutic target.</p>
							<p><a class="btn" data-toggle="collapse" data-target="#man1">Abstract &raquo;</a></p>
						</div>
					</div>
		    </div>
				<div class="verticalLine"></div>
		    <div class="col">
				<div id="wrapper">
					<div id="content">
						<img class="research" src="./static/research.svg" alt="research" width=5 height=5/><h4 id="manuscript-title">Assembly and interrogation of Alzheimer's disease genetic networks reveal novel regulators of progression.</h4><p>Aubry S, Shin W, Crary JF, Lefort R, Qureshi YH, Lefebvre C, Califano A, Shelanski ML.</p>
						<a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0120352" target="_blank" rel="noopener noreferrer">PLoS One. 2015 Mar 17;10(3):e0120352</p></a>
						<p class="collapse" id="man2">Alzheimer's disease (AD) is a complex multifactorial disorder with poorly characterized pathogenesis. Our understanding of this disease would thus benefit from an approach that addresses this complexity by elucidating the regulatory networks that are dysregulated in the neural compartment of AD patients, across distinct brain regions. Here, we use a Systems Biology (SB) approach, which has been highly successful in the dissection of cancer related phenotypes, to reverse engineer the transcriptional regulation layer of human neuronal cells and interrogate it to infer candidate Master Regulators (MRs) responsible for disease progression. Analysis of gene expression profiles from laser-captured neurons from AD and controls subjects, using the Algorithm for the Reconstruction of Accurate Cellular Networks (ARACNe), yielded an interactome consisting of 488,353 transcription-factor/target interactions. Interrogation of this interactome, using the Master Regulator INference algorithm (MARINa), identified an unbiased set of candidate MRs causally responsible for regulating the transcriptional signature of AD progression. Experimental assays in autopsy-derived human brain tissue showed that three of the top candidate MRs (YY1, p300 and ZMYM3) are indeed biochemically and histopathologically dysregulated in AD brains compared to controls. Our results additionally implicate p53 and loss of acetylation homeostasis in the neurodegenerative process. This study suggests that an integrative, SB approach can be applied to AD and other neurodegenerative diseases, and provide significant novel insight on the disease progression.</p>
						<p><a class="btn" data-toggle="collapse" data-target="#man2">Abstract &raquo;</a></p>
					</div>
				</div>
		   </div>
		  </div>
		</div>
		<div class="container">
		<div class="row">
			<div class="col">
				<div id="wrapper">
					<div id="content">
						<img class="research" src="./static/research.svg" alt="research" width=5 height=5/><h4 id="manuscript-title">Reversing synapse loss in Alzheimer's disease: Rho-guanosine triphosphatases and insights from other brain disorders.</h4><p>Lefort R.</p>
						<a href="https://link.springer.com/article/10.1007%2Fs13311-014-0328-4" target="_blank" rel="noopener noreferrer">Neurotherapeutics. 2015 Jan;12(1):19-28</p></a>
						<p class="collapse" id="man3">Alzheimer's disease (AD) is a monumental public health crisis with no effective cure or treatment. To date, therapeutic strategies have focused almost exclusively on upstream signaling events in the disease, namely on β-amyloid and amyloid precursor protein processing, and have, unfortunately, yielded few, if any, promising results. An alternative approach may be to target signaling events downstream of β-amyloid and even tau. However, with so many pathways already linked to the disease, understanding which ones are "drivers" versus "passengers" in the pathogenesis of the disease remains a tremendous challenge. Given the critical roles of Rho-guanosine triphosphatases (GTPases) in regulating the actin cytoskeleton and spine dynamics, and the strong association between spine abnormalities and cognition, it is not surprising that mutations in a number of genes involved in Rho-GTPase signaling have been implicated in several brain disorders, including schizophrenia and autism. And now, there is mounting literature implicating Rho-GTPase signaling in AD pathogenesis as well. Here, I review this evidence, with a particular emphasis on the regulators of Rho-GTPase signaling, namely guanine nucleotide exchange factors and GTPase-activating proteins. Several of these have been linked to various aspects of AD, and each offers a novel potential therapeutic target for AD.</p>
						<p><a class="btn" data-toggle="collapse" data-target="#man3">Abstract &raquo;</a></p>
					</div>
				</div>
			</div>
			<div class="verticalLine"></div>
			<div class="col">
			<div id="wrapper">
				<div id="content">
					<img class="research" src="./static/research.svg" alt="research" width=5 height=5/><h4 id="manuscript-title">The Aβ₁₋₄₂ peptide regulates microtubule stability independently of tau.</h4><p>Pianu B, Lefort R, Thuiliere L, Tabourier E, Bartolini F.</p>
					<a href="http://jcs.biologists.org/cgi/pmidlookup?view=long&pmid=24424028" target="_blank" rel="noopener noreferrer">J Cell Sci. 2014 Mar 1;127(Pt 5):1117-27</p></a>
					<p class="collapse" id="man4">Interference with microtubule stability by beta-amyloid peptide (Aβ) has been shown to disrupt dendritic function and axonal trafficking, both early events in Alzheimer's disease. However, it is unclear whether Aβ regulation of microtubule dynamics can occur independently of its action on tau. RhoA has been implicated in neurotoxicity by Aβ but the mechanism by which this activation generates cytoskeletal changes is also unclear. We found that oligomeric Aβ1-42 induced the formation of stable detyrosinated microtubules in NIH3T3 cells and this function resulted from the activation of a RhoA-dependent microtubule stabilization pathway regulated by integrin signaling and the formin mDia1. Induction of microtubule stability by Aβ was also initiated by dimerization of APP and required caspase activity, two previously characterized regulators of neurotoxicity downstream of Aβ. Finally, we found that this function was conserved in primary neurons and abolished by Rho inactivation, reinforcing a link between induction of stable detyrosinated microtubules and neuropathogenesis by Aβ. Our study reveals a novel activity of Aβ on the microtubule cytoskeleton that is independent of tau and associated with pathways linked to microtubule stabilization and Aβ-mediated neurotoxicity.</p>
					<p><a class="btn" data-toggle="collapse" data-target="#man4">Abstract &raquo;</a></p>
				</div>
			</div>
		 </div>
		 <div class="container">
		 <div class="row">
			 <div class="col">
				 <div id="wrapper">
					 <div id="content">
						 <img class="research" src="./static/research.svg" alt="research" width=5 height=5/><h4 id="manuscript-title">Caspase-2 is required for dendritic spine and behavioural alterations in J20 APP transgenic mice.</h4><p>Pozueta J, Lefort R, Ribe EM, Troy CM, Arancio O, Shelanski M.</p>
						 <a href="https://www.nature.com/articles/s42003-019-0350-5" target="_blank" rel="noopener noreferrer">Nat Commun. 2013;4:1939</p></a>
						 <p class="collapse" id="man5">Caspases have critical roles in Alzheimer's disease pathogenesis. Here we show that caspase-2 is required for the cognitive decline seen in human amyloid precursor protein transgenic mice (J20). The age-related changes in behaviour and dendritic spine density observed in these mice are absent when they lack caspase-2, in spite of similar levels of amyloid beta (Aβ) deposition and inflammation. A similar degree of protection is observed in cultured hippocampal neurons lacking caspase-2, which are immune to the synaptotoxic effects of Aβ. Our studies suggest that caspase-2 is a critical mediator in the activation of the RhoA/ROCK-II signalling pathway, leading to the collapse of dendritic spines. We propose that this is controlled by an inactive caspase-2/RhoA/ROCK-II complex localized in dendrites, which dissociates in the presence of Aβ, allowing for their activation and entry in the spine. These findings directly implicate caspase-2 as key driver of synaptic dysfunction in Alzheimer's disease and offer novel therapeutic targets.</p>
						 <p><a class="btn" data-toggle="collapse" data-target="#man5">Abstract &raquo;</a></p>
					 </div>
				 </div>
			 </div>
			 <div class="verticalLine"></div>
			 <div class="col">
			 <div id="wrapper">
				 <div id="content">
					 <img class="research" src="./static/research.svg" alt="research" width=5 height=5/><h4 id="manuscript-title">Cross-linking of cell surface amyloid precursor protein leads to increased β-amyloid peptide production in hippocampal neurons: implications for Alzheimer's disease.</h4><p>Lefort R, Pozueta J, Shelanski M.</p>
					 <a href="http://www.jneurosci.org/cgi/pmidlookup?view=long&pmid=22855816" target="_blank" rel="noopener noreferrer">J Neurosci. 2012 Aug 1;32(31):10674-85</p></a>
					 <p class="collapse" id="man6">The accumulation of the β-amyloid peptide (Aβ) in Alzheimer's disease (AD) is thought to play a causative role in triggering synaptic dysfunction in neurons, leading to their eventual demise through apoptosis. Aβ is produced and secreted upon sequential cleavage of the amyloid precursor protein (APP) by β-secretases and γ-secretases. However, while Aβ levels have been shown to be increased in the brains of AD patients, little is known about how the cleavage of APP and the subsequent generation of Aβ is influenced, or whether the cleavage process changes over time. It has been proposed that Aβ can bind APP and promote amyloidogenic processing of APP, further enhancing Aβ production. Proof of this idea has remained elusive because a clear mechanism has not been identified, and the promiscuous nature of Aβ binding complicates the task of demonstrating the idea. To work around these problems, we used an antibody-mediated approach to bind and cross-link cell-surface APP in cultured rat primary hippocampal neurons. Here we show that cross-linking of APP is sufficient to raise the levels of Aβ in viable neurons with a concomitant increase in the levels of the β-secretase BACE1. This appears to occur as a result of a sorting defect that stems from the caspase-3-mediated inactivation of a key sorting adaptor protein, namely GGA3, which prevents the lysosomal degradation of BACE1. Together, our data suggest the occurrence of a positive pathogenic feedback loop involving Aβ and APP in affected neurons possibly allowing Aβ to spread to nearby healthy neurons.</p>
					 <p><a class="btn" data-toggle="collapse" data-target="#man6">Abstract &raquo;</a></p>
				 </div>
			 </div>
			</div>
		 </div>
		 <a href="https://pubmed.ncbi.nlm.nih.gov/?term=roger+lefort&sort=date" target="_blank" rel="noopener noreferrer"<button type="button" class="btn btn-light">Full list of publications</button></a>
		</div>
		</div>
		</div>
		</article>
			<hr>
		<article>
			<div id="projects"></div>
			<h2>Projects</h2>

			<div class="card-columns">

				<div class="card">
					<a href="https://healthy-heart-webapp.herokuapp.com/" target="_blank" rel="noopener noreferrer">
						<img class="card-img-top" src="./static/heart_beating.gif" alt="beating heart"></a>
					<div class="card-body">
						<h5 class="card-title">Predicting Cardiovascular Disease Risk</h5>
						<p class="card-text collapse" id="heart">People with CVD or those who are at high cardiovascular risk (due to the presence of one or more risk factors such as hypertension, diabetes, hyperlipidaemia or other comorbidities) need early detection and management wherein a machine learning model can be of great help.</p>
						<p><a class="btn" data-toggle="collapse" data-target="#heart">Read More &raquo;</a></p>
					</div>
					<div class="card-footer">
						<a href="https://github.com/tiroger/heart-disease" target="_blank" rel="noopener noreferrer"><small class="text-muted">Github:
								www.github.com/tiroger/heart-disease</small></a>
					</div>
				</div>

  		<div class="card">
			<a href="https://share.streamlit.io/tiroger/nyc-vehicle-collisions/main/app.py" target="_blank" rel="noopener noreferrer">
    		<img class="card-img-top" src="./static/images/accident.gif" alt="NYPD"></a>
    		<div class="card-body">
      			<h5 class="card-title">NYC Vehicle Collisions Dashboard</h5>
      			<p class="card-text collapse" id="nypd">In 2011 the New York City Council passed Local Law #12 requiring the NYPD to collect and make available to the public records of all accidents involving motor vehicles. A police report (MV104-AN) is required to be filled out for collisions where someone is injured or killed, or where there is at least $1000 worth of damage. Each report represents a unique collision and includes date and time, location(city, borough, address, zip code, latitude and longitude), number of injuries and/or deaths, contributing factors, and information about the vehicles involved. The information is available in a variety of formats, including downloadable json and csv files, and through the Socrata Open Data API.</br>
				I used the data to identify trends and to test different hypotheses.</p>
				<p><a class="btn" data-toggle="collapse" data-target="#nypd">Read More &raquo;</a></p>
    		</div>
    		<div class="card-footer">
      			<a href="#"><small class="text-muted">Github: https://github.com/tiroger/nyc-vehicle-collisions</small></a>
    		</div>
		  </div>

  			<div class="card">
				<a href="http://biodiversity-and-trail-maps-of.herokuapp.com" target="_blank" rel="noopener noreferrer">
    			<img class="card-img-top" src="./static/bear_walking.gif" alt="National Parks Logo"></a>
    		<div class="card-body">
      		<h5 class="card-title">Visualizing National Park Biodiversity</h5>
      		<p class="card-text collapse" id="park">The National Park Service publishes a database of animal and plant species identified in individual national parks and verified by evidence — observations, vouchers, or reports that document the presence of a species in a park. All park species records are available to the public on the National Park Species portal; exceptions are made for sensitive, threatened, or endangered species when widespread distribution of information could pose a risk to the species in the park.<br>
			Here I created a dashboard to show the diversity of the flora and fauna in each National Park.</p>
			<p><a class="btn" data-toggle="collapse" data-target="#park">Read More &raquo;</a></p>
    		</div>
    		<div class="card-footer">
      		<a href="https://github.com/tiroger/Biodiversity-and-Trail-Maps-of-National-Parks" target="_blank" rel="noopener noreferrer"><small class="text-muted">Github: github.com/tiroger/Biodiversity-and-Trail-Maps-of-National-Parks</small></a>
    		</div>
  			</div>	  

		  <div class="card">
			<a href="https://coronavirus-cases.herokuapp.com/" target="_blank" rel="noopener noreferrer">
				<img class="card-img-top" src="./static/coronavirus.gif" alt="coronavirus"></a>
			<div class="card-body">
				<h5 class="card-title">Visuazing the COVID-19 Pandemic with Plotly and Dash</h5>
				<p class="card-text collapse" id="coronavirus">This dashboard was built with Plotly and Dash using data from the Johns Hopkins Center for Systems Science and
				Engineering (JHU/CSSE) github repository. The data is updated nightly and changes are reflected here upon app launch.</p>
				<p><a class="btn" data-toggle="collapse" data-target="#coronavirus">Read More &raquo;</a></p>
			</div>
			<div class="card-footer">
				<a href="https://github.com/tiroger/coronavirus-tracker" target="_blank" rel="noopener noreferrer"><small class="text-muted">Github:
						www.github.com/tiroger/coronavirus-tracker</small></a>
			</div>
		</div>

		  <div class="card">
					<a href="https://tiroger.github.io/nyc_rats/index.html" target="_blank" rel="noopener noreferrer">
					<img class="card-img-top" src="./static/images/nyc_rat.gif" alt="Card image cap"></a>
					<div class="card-body">
						<h5 class="card-title">Analyzing NYC Rat Population with Tableau</h5>
						<p class="card-text collapse" id="rats">Rats in New York City are prevalent, as in many densely populated areas. For a long time, the exact number of rats in New York City was unknown, and a common urban legend was that there were up to four times as many rats as people. In 2014, however, scientists more accurately measured the entire city's rat population to be approximately only 25% of the number of humans; i.e., there were approximately 2 million rats to New York's 8.4 million people at the time of the study.</br>

						Data is from 2010-Sept 16th, 2017 and includes date, location (lat/lon), type of structure, borough, and community
						board.

						<br>
						Questions

						<br>Q1 - Which Borough and zip codes have the most rat sightings?

						<br>Q2 - Is there a correlation with zip code population?

						<br>Q3 - Have rat sightings increased over the years?

						<br>Q4 - Is there a seasonal pattern to rat sightings?

						<p><a class="btn" data-toggle="collapse" data-target="#rats">Read More &raquo;</a></p>

					</div>
					<div class="card-footer">
						<a href="https://github.com/tiroger/nyc_rats" target="_blank" rel="noopener noreferrer"><small class="text-muted">Github: github.com/tiroger/nyc_rats</small></a>
					</div>
				</div>

			<div class="card">
				<a href="https://tiroger.github.io/Global-Earthquake-Data-Visualization/" target="_blank" rel="noopener noreferrer">
				<img class="card-img-top" src="./static/images/earthquake.gif" alt="Earquake"></a>
				<div class="card-body">
					<h5 class="card-title">Visualizing Worldwide Earthquakes with leaflet.js</h5>
					<p class="card-text collapse" id="earthquake">The USGS is responsible for providing scientific data about natural hazards, the health of our ecosystems and environment and the impacts of climate and land-use change. Their scientists develop new methods and tools to supply timely, relevant, and useful information about the Earth and its processes.

					<br>The USGS provides earthquake data in a number of different formats, updated every 5 minutes - USGS GeoJSON Feed. For
					example, when you click on the data set 'All Earthquakes from the Past 7 Days', you are given a JSON representation of
					that data. I used the URL of this JSON to pull in the data for visualization.

					<br>I created a map using Leaflet that plots all of the earthquakes from the data set based on their longitude and latitude.

					<br>- The data markers reflect the magnitude of the earthquake in their size and color. Earthquakes with higher magnitudes
					appear larger and darker in color.

					<br>- It includes popups that provide additional information about the earthquake when a marker is clicked.

					<br>- Included is a legend that provides context for your map data.

					<br>There is also a second data set on the map that illustrates the relationship between tectonic plates and seismic
					activity. I pulled in a second data set and visualized it along side the original set of data. Data on tectonic plates
					can be found at https://github.com/fraxen/tectonicplates.</p>
					<p><a class="btn" data-toggle="collapse" data-target="#earthquake">Read More &raquo;</a></p>
				</div>
				<div class="card-footer">
					<a href="https://github.com/tiroger/Global-Earthquake-Data-Visualization" target="_blank" rel="noopener noreferrer"><small class="text-muted">Github: github.com/tiroger/Global-Earthquake-Data-Visualization</small></a>
				</div>
				</div>

				<div class="card">
					<a href="http://www.nflplaypredictions.com/"target="_blank" rel="noopener noreferrer" >
    				<img class="card-img-top" src="./static/standard-deviation.gif" alt="NFL Plays"></a>
    				<div class="card-body">
      					<h5 class="card-title">A Machine Learning Approach to Predicting NFL Plays with Scikit-Learn</h5>
						<p class="card-text collapse" id="nfl">In the National Football League (NFL), being able to accurately predict which play your oppent will call given a set of circumstances would be an invaluable tool, and teams devote many resources towards gaining insights into an opponent's tendencies on the field. One area of immense interest is assessing a team's propensity to run or pass the ball in a given situation, which for a defense can inform critical decision-making about strategy, personnel, and positioning on the field, decisions that have a profound impact on the outcome of a game.
						<br>
						In this project, I take a data-driven approach to classifying offensive play types and examine whether tools from
						machine learning can offer value in characterizing offensive tendencies.</p>
						<p><a class="btn" data-toggle="collapse" data-target="#nfl">Read More &raquo;</a></p>
    				</div>
    				<div class="card-footer">
      					<a href="https://github.com/tiroger/Predicting_NFL_Stats_ML" target="_blank" rel="noopener noreferrer"><small class="text-muted">Github: www.github.com/tiroger/Prediciting_NFL_Stats_ML</small></a>
    				</div>
  				</div>

				<div class="card view">
				<a href="https://github.com/tiroger/predicting-loan-default" target="_blank" rel="noopener noreferrer">
				<img class="card-img-top" src="./static/images/loan.gif" alt="Loan Defaults"></a>
				<div class="card-body mask flex-center rgba-red-strong">
					<h5 class="card-title">Using Machine Learning to Predict Loan Defaults</h5>
					<p class="card-text collapse" id="loans">LendingClub positions itself as a social networking service and sets up opportunities for members to identify group affinities, based on a theory that borrowers are less likely to default to lenders with whom they have affinities and social relationships. It developed an algorithm called LendingMatch to identify these common relationship factors (e.g.geographic location, educational and professional background).
					<br>

					It offers the algorithm as a search tool for investors to find Notes they would like to purchase, using borrower and loan attributes such as the length of a loan term, target weighted average interest rate, borrower credit score employment tenure, home ownership status, and others. To reduce default risk, LendingClub focuses on high-credit-worthy borrowers, declining approximately 90% of the loan applications it received as of 2012 and assigning higher interest rates to riskier borrowers within its credit criteria.<br>
					<br>
					<strong>Project Goal<strong>
					<br>
					Can we use Machine Learning to build a model that predicts whether a borrower will default on a loan, that matches or exceeds LendingClub's own algorithm? Can we identify the most important features that determine the lean outcome?</p>
					</p><a class="btn" data-toggle="collapse" data-target="#loans">Read More &raquo;</a></p>
				</div>
				<div class="card-footer">
					<a href="https://github.com/tiroger/predicting-loan-default" target="_blank" rel="noopener noreferrer"><small class="text-muted">Github: github.com/tiroger/predicting-loan-default</small></a>
				</div>
				</div>

	</div>

			<!-- <br> -->

			<!-- <div class="card-deck">

				<div class="card">
					<a href="#">
					<img class="card-img-top" src="https://media2.giphy.com/media/l4FGoECUgPTkFX7La/giphy.gif/" alt="mouse"></a>
					<div class="card-body">
						<h5 class="card-title">Predicting Mouse Phenotypes using Machine Learning</h5>
						<p class="card-text">TBD.</p>
					</div>
					<div class="card-footer">
						<a href="#"><small class="text-muted">Github: TBD</small></a>
					</div>
				</div>

				<div class="card">
				<a href="http://biodiversity-and-trail-maps-of.herokuapp.com">
				<img class="card-img-top" src="https://www.americannursetoday.com/wp-content/uploads/2014/07/brain-tree-dementia.jpg" alt="Card image cap"></a>
				<div class="card-body">
					<h5 class="card-title">Deep Learning for Alzheimer's Disease Classification</h5>
					<p class="card-text">This card has supporting text below as a natural lead-in to additional content.</p>
				</div>
				<div class="card-footer">
					<a href="#"><small class="text-muted">Github: TBD</small></a>
				</div>
				</div> -->

			<!-- <div class="card view">
				<a href="https://www.w3schools.com">
				<img class="card-img-top" src="https://media2.giphy.com/media/l4FGoECUgPTkFX7La/giphy.gif" alt="NFL Machine"></a>
				<div class="card-body mask flex-center rgba-red-strong">
					<h5 class="card-title">Mouse Phenotype Classification using Machine Learning</h5>
					<p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This card has even longer content than the first to show that equal height action.</p>
				</div>
				<div class="card-footer">
					<small class="text-muted">Last updated 3 mins ago</small>
				</div>
				</div> -->
			<!-- </div> -->
			<!-- <br> -->
			<!-- <div class="card-columns"> -->

				<!-- <div class="card">
					<a href="https://docs.google.com/presentation/d/1-GCz8RwRkCS_pjHM0wbCNg7CW8YoLZajcYTy4QvtO3g/edit?usp=sharing">
					<img class="card-img-top" src="https://i.pinimg.com/originals/52/48/9f/52489f3ebedf90b573fb2fb674e4c076.gif" alt="Card image cap"></a>
					<div class="card-body">
						<h5 class="card-title">Analyzing NYC Rat Population</h5>
						<p class="card-text collapse" id="rats">Rats in New York City are prevalent, as in many densely populated areas. For a long time,
						the exact number of rats in New York City was unknown, and a common urban legend was that there were up to four times as many rats as people. In 2014, however, scientists more accurately measured the entire city's rat population to be approximately only 25% of the number of humans; i.e., there were approximately 2 million rats to New York's 8.4 million
						people at the time of the study.</br>

						New York City rodent complaints can be made online, or by dialing 3-1-1, and the New York City guide (Preventing Rats on Your Property) discusses how the New York City Health Department inspects private and public properties for rats.
						Property owners that fail inspections receive a Commissioner's Order and have five days to correct the problem. If after
						five days the property fails a second inspection, the owner receives a Notice of Violation and can be fined. The
						property owner is billed for any clean-up or extermination carried out by the Health Department. Data is from 2010-Sept 16th, 2017 and includes date, location (lat/lon), type of structure, borough, and community
						board.

						<br>
						Questions

						<br>Q1 - Which Borough and zip codes have the most rat sightings?

						<br>Q2 - Is there a correlation with zip code population and restaurant density?

						<br>Q3 - Have rat sightings increased over the years?

						<br>Q3 - Is there a seasonal pattern to rat sightings?

						<br>Q4 - Is there a correlation between rat sightings and climate parameters?

						<br>Q5 - Do fluctiuations in NYC weather correlate with rat sightings?</p>
						<p><a class="btn" data-toggle="collapse" data-target="#rats">Read More &raquo;</a></p>


					</div>
					<div class="card-footer">
						<a href="https://github.com/tiroger/nyc_rats"><small class="text-muted">Github: github.com/tiroger/nyc_rats</small></a>
					</div>
				</div> -->

				<!-- <div class="card">
				<a href="https://tiroger.github.io/Global-Earthquake-Data-Visualization/">
				<img class="card-img-top" src="https://cdn.images.express.co.uk/img/dynamic/151/590x/california-1191568.jpg?r=1571240067202" alt="Earquake"></a>
				<div class="card-body">
					<h5 class="card-title">Visualizing Worldwide Earthquake data</h5>
					<p class="card-text collapse" id="earthquake">The USGS is responsible for providing scientific data about natural hazards, the health of our ecosystems and
					environment and the impacts of climate and land-use change. Their scientists develop new methods and tools to supply
					timely, relevant, and useful information about the Earth and its processes.

					<br>The USGS provides earthquake data in a number of different formats, updated every 5 minutes - USGS GeoJSON Feed. For
					example, when you click on the data set 'All Earthquakes from the Past 7 Days', you are given a JSON representation of
					that data. I used the URL of this JSON to pull in the data for visualization.

					<br>I created a map using Leaflet that plots all of the earthquakes from the data set based on their longitude and latitude.

					<br>- The data markers reflect the magnitude of the earthquake in their size and color. Earthquakes with higher magnitudes
					appear larger and darker in color.

					<br>- It includes popups that provide additional information about the earthquake when a marker is clicked.

					<br>- Included is a legend that provides context for your map data.

					<br>There is also a second data set on the map that illustrates the relationship between tectonic plates and seismic
					activity. I pulled in a second data set and visualized it along side the original set of data. Data on tectonic plates
					can be found at https://github.com/fraxen/tectonicplates.</p>
					<p><a class="btn" data-toggle="collapse" data-target="#earthquake">Read More &raquo;</a></p>
				</div>
				<div class="card-footer">
					<a href="https://github.com/tiroger/Global-Earthquake-Data-Visualization"><small class="text-muted">Github: github.com/tiroger/Global-Earthquake-Data-Visualization</small></a>
				</div>
				</div> -->

			<!-- <div class="card view">
				<a href="https://www.w3schools.com">
				<img class="card-img-top" src="https://media2.giphy.com/media/l4FGoECUgPTkFX7La/giphy.gif" alt="NFL Machine"></a>
				<div class="card-body mask flex-center rgba-red-strong">
					<h5 class="card-title">Mouse Phenotype Classification using Machine Learning</h5>
					<p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This card has even longer content than the first to show that equal height action.</p>
				</div>
				<div class="card-footer">
					<small class="text-muted">Last updated 3 mins ago</small>
				</div>
				</div> -->
			<!-- </div> -->
			</article>
			<hr>

			<div id="onlinepresence">
			<h2>Online Presence</h2>
			<p>

			<a href="https://github.com/tiroger" target="_blank" rel="noopener noreferrer"><img src="./static/socicon_github.svg" alt="github" width=24 height=24/></a>
			<a href="https://www.linkedin.com/in/roger-lefort/" target="_blank" rel="noopener noreferrer"><img src="./static/socicon_linkedin.svg" alt="linkedin"  width=24 height=24/></a>
			<a href="https://twitter.com/tiroger" target="_blank" rel="noopener noreferrer"><img src="./static/socicon_twitter.svg" alt="twitter"  width=24 height=24/></a>
			<a href="https://orcid.org/0000-0002-8885-606X/print" target="_blank" rel="noopener noreferrer"><img src="./static/images/socicon_orcid.svg" alt="ORCID"  width=24 height=24/></a>
			<a href="https://www.strava.com/athletes/644338" target="_blank" rel="noopener noreferrer"><img src="./static/images/socicon_strava.svg" alt="Strava"  width=24 height=24/></a>
			<div>
			<iframe height='160' width='300' frameborder='0' allowtransparency='true' scrolling='no' src='https://m.strava.com/athletes/644338/activity-summary/53c89f1acdf2bf69a8bc937e3793e9bfb56d64be'></iframe>
			</div>
			

			</p>

			</div>
			</article>

			<!-- Button to scroll to sop of page -->
			<button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>
			<script>
			//Get the button
			var mybutton = document.getElementById("myBtn");

			// When the user scrolls down 20px from the top of the document, show the button
			window.onscroll = function() {scrollFunction()};

			function scrollFunction() {
  		if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
    			mybutton.style.display = "block";
  				} else {
    			mybutton.style.display = "none";
  			}
			}

			// When the user clicks on the button, scroll to the top of the document
			function topFunction() {
  			document.body.scrollTop = 0;
  			document.documentElement.scrollTop = 0;
				}
			</script>
			</section>
			<footer>
				<div>
				<p><a href="https://twitter.com/share" class="twitter-share-button" data-via="tiroger" data-hashtags="resume">Share this resume via Twitter</a>
				<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script></p>
				<p>Copyright 2019 &copy; Roger Lefort,&nbsp;<script>
				var username = "roger";
				var hostname = "cyclingscientist.com";
				var linktext = username + "@" + hostname ;
				document.write("<a href='" + "mail" + "to:" + username + "@" + hostname + "'>" + linktext + "</a><br/>");
				</script></p>
			</div>
			</footer>

		<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.4.11/d3.min.js" charset="utf-8"></script>
		<script src="./static/js/timeline.js" charset="utf-8"></script>
		<script src="./static/js/skills.js" charset="utf-8"></script>
		   <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>

    	<!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="https://code.jquery.com/jquery-3.1.0.js"></script>

		<!-- <script src="js/jquery-1.11.1.min.js"></script> -->


	<!-- Bootstrap  -->
	<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
</body>
</html>
